hbase-spark/src/test/scala/geotrellis/spark/HBaseTestEnvironment.scala

```diff
@@ -14,7 +14,10 @@
  * limitations under the License.
  */
 
-package geotrellis.store.hbase
```

Comment:
We should make the `TestEnvironment` traits available from our local test packages and break off spark-requiring portions to live inside `geotrellis.store.spark` tests